distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -287,4 +292,10 @@ start -p 8994
 -----
 +
 Then click `Next >`.
-. Review the settings and click `Finish`. Additional settings can be made by right clicking the task and updating the Properties. It may be necessary to update the Security Options under the task Properties to `Run with highest privileges`.
\ No newline at end of file
+. Review the settings and click `Finish`. Additional settings can be made by right clicking the task and updating the Properties. It may be necessary to update the Security Options under the task Properties to `Run with highest privileges` or even setting user to "SYSTEM".
+Additionally, the process can be set to restart if it fails under the Properties > Settings tab. Depending on the system it may also make sense to delay the process from starting for a few minutes until the machine has fully booted. This can be adjusted under Properties > Triggers > Edit > Advanced Settings > Delay Task
+
+[NOTE]
+====
+If either process is shutdown (${branding} or Solr) the other will continue running.
```

Comment:
❓ I thought I have seen shutdown DDF also shutting down Solr. Am I confused on this statement?

Reply:
That is true under the "normal" use case when the system is configured to manage Solr alongside DDF. This note is in the section where the system is setup as a process that automatically boots. In that case the DDF is not managing Solr and each of them can be stopped independently.